test/unit/3dtileslayerprocessbatchtable.js

```diff
@@ -0,0 +1,44 @@
+import assert from 'assert';
+import C3DTilesLayer, { $3dTilesExtensions } from 'Layer/C3DTilesLayer';
+import BatchTableHierarchyExtensionParser from 'Parser/BatchTableHierarchyExtensionParser';
+import C3DTilesSource from 'Source/C3DTilesSource';
+import View from 'Core/View';
+import GlobeView from 'Core/Prefab/GlobeView';
+import HttpsProxyAgent from 'https-proxy-agent';
+import Coordinates from 'Core/Geographic/Coordinates';
+import Renderer from './bootstrap';
+
+describe('3Dtiles batch table', function () {
+    const renderer = new Renderer();
+
+    const p = {
+        coord: new Coordinates('EPSG:4326', -75.61349, 40.044259),
+        range: 200,
+        tilt: 10,
+        heading: -145,
+    };
+    const viewer = new GlobeView(renderer.domElement, p, { renderer, noControls: true });
+
+
+    const threedTilesLayerBTHierarchy = new C3DTilesLayer(
+        '3d-tiles-bt-hierarchy', {
+            name: 'BTHierarchy',
+            source: new C3DTilesSource({
+                url: 'https://raw.githubusercontent.com/AnalyticalGraphicsInc/cesium/master/Apps/SampleData/Cesium3DTiles/Hierarchy/BatchTableHierarchy/tileset.json',
```

Comment:
Couldn't we store file like this on the repository ? That way we are sure we always have access to it. I know this isn't the only place we are doing this, so I'm more asking as a general question rather than for this PR in particular.